setup.py

```diff
@@ -3,8 +3,8 @@
 
 ENTRY_POINTS = {
     'pipeline.handlers': [
-        'AatamsDmHandler = aodndata.aatams.aatams_dm:AatamsDmHandler',
-        'AatamsNrtHandler = aodndata.aatams.aatams_nrt:AatamsNrtHandler',
+        'AatamsSattagQcDmHandler = aodndata.aatams.aatams_sattag:AatamsSattagQcDmHandler',
+        'AatamsSattagNrtHandler = aodndata.aatams.aatams_sattag:AatamsSattagNrtHandler',
```

Comment:
the handler has to be renamed here to match your changes

Reply:
Arg, always forgetting these bits.